### DISCUSSION TITLE
HFS Newie

### DISCUSSION THREAD

**Discussion by 4AllMankind-4AM** [rejetto/hfs#726] [General]
I'm new to hfs so excuse the newbie question. I believe I have setup HFS properly. When I start the program on my windows 11 PC as the administrator the home menu shows green & the server is working. 
![server](https://github.com/user-attachments/assets/52ba630d-29d5-4b7a-94da-4f7591d0b09a)
I then move to the internet tab to make sure everything else was green & working only to notice the server with port 1080 was green & the internet server is responding as it should, but the router is showing port unknown & a operation failed when I click on it.

![HFS](https://github.com/user-attachments/assets/463f16f1-2264-4c11-9223-d2c89ffbf297)

I've visited a couple of websites to check if my port 1080 is open and both sites I've visited shows that it is. Nonetheless I still portforwarded 1080 on my bell home hub 3000 router & still can't get hfs to except any visitors outside of my home network.
![portforward](https://github.com/user-attachments/assets/575ed696-a333-470c-bbed-58523a9c3106)

I'm not sure what I'm doing wrong & can sure use some help.

**Comment by SanokKule** [comment]
I can access your server just fine, not sure what might be wrong here

**Reply by 4AllMankind-4AM** [reply]
> I can access your server just fine, not sure what might be wrong here

Thank you apparently my visitors were using https instead of http.

**Comment by TEA-Time1** [comment]
I was able to access it too. Make sure your visitors are using exactly what's in your second screenshot. The "http" part can't be "https", or the browser will expect an encrypted connection.  And the ":1080" at the end can't be left off because it's a non-standard port (which is fine), so the browser has to be told what port to use.

**Reply by 4AllMankind-4AM** [reply]
Yup! you're right they were using https thank you it's all good now.

**Reply by TEA-Time1** [reply]
Cool, you're welcome. 👍
